### ISSUE TITLE
Build failure on Ubuntu 10.04

### ISSUE THREAD

**Issue by kenetik** [vasi/pixz#28]
What am I missing or doing wrong?

```
[~/downloads/pixz-1.0.2]# make
gcc  -g -O0 -std=gnu99 -Wall -Wno-unknown-pragmas -DPIXZ_VERSION='"1.0.2"'      -c -o common.o common.c
common.c: In function ‘find_file_index’:
common.c:119: error: ‘lzma_index_iter’ undeclared (first use in this function)
common.c:119: error: (Each undeclared identifier is reported only once
common.c:119: error: for each function it appears in.)
common.c:119: error: expected ‘;’ before ‘iter’
common.c:120: warning: implicit declaration of function ‘lzma_index_iter_init’
common.c:120: error: ‘iter’ undeclared (first use in this function)
common.c:122: warning: implicit declaration of function ‘lzma_index_iter_locate’
common.c: In function ‘next_index’:
common.c:329: warning: implicit declaration of function ‘lzma_index_stream_flags’
common.c:331: warning: implicit declaration of function     ‘lzma_index_stream_padding’
common.c: In function ‘decode_index’:
common.c:344: error: too few arguments to function ‘lzma_index_cat’
make: *** [common.o] Error 1
```

**Comment by wookietreiber** [comment]
Ubuntu 10.04 is out of official support:

> Canonical provided support for the desktop version of Ubuntu 10.04 until 9 May 2013 and the server version until 30 April 2015.

@vasi are you with me on closing this issue?

**Comment by vasi** [comment]
Hmm, I don't think too many people are using liblzma 4.999. But I think we did once support this version? Maybe let's leave it open as a low-priority bug—if someone actually has an old liblzma and comes along with a fix, I'll take it :)

**Comment by mietek** [comment]
FWIW, the issue is no longer relevant to me.

**Comment by tomck** [comment]
xz (XZ Utils) 4.999.9beta
liblzma 4.999.9beta

Still the "latest" for CentOS 6 series, which is (unfortunately) widely used in cases where pixz would excel 😛 

`make  all-recursive
make[1]: Entering directory `/root/pixz-1.0.6'
Making all in src
make[2]: Entering directory `/root/pixz-1.0.6/src'
gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I..     -pthread -Wall -Wno-unknown-pragmas -g -O2 -MT pixz-common.o -MD -MP -MF .deps/pixz-common.Tpo -c -o pixz-common.o `test -f 'common.c' || echo './'`common.c
common.c: In function ‘find_file_index’:
common.c:120: error: ‘lzma_index_iter’ undeclared (first use in this function)
common.c:120: error: (Each undeclared identifier is reported only once
common.c:120: error: for each function it appears in.)
common.c:120: error: expected ‘;’ before ‘iter’
common.c:121: warning: implicit declaration of function ‘lzma_index_iter_init’
common.c:121: error: ‘iter’ undeclared (first use in this function)
common.c:123: warning: implicit declaration of function ‘lzma_index_iter_locate’
common.c: In function ‘next_index’:
common.c:339: warning: implicit declaration of function ‘lzma_index_stream_flags’
common.c:341: warning: implicit declaration of function ‘lzma_index_stream_padding’
common.c: In function ‘decode_index’:
common.c:357: error: too few arguments to function ‘lzma_index_cat’
make[2]: *** [pixz-common.o] Error 1
make[2]: Leaving directory `/root/pixz-1.0.6/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/root/pixz-1.0.6'
make: *** [all] Error 2
`
